internal/core/textlayout/linebreak_unicode.rs

```diff
@@ -19,11 +19,7 @@ impl<'a> LineBreakIterator<'a> {
         let iterator = unicode_linebreak::linebreaks(text).filter(|(offset, opportunity)| {
             // unicode-linebreaks emits a mandatory break at the end of the text. We're not interested
             // in that.
-            if *offset == text.len() && matches!(opportunity, BreakOpportunity::Mandatory) {
-                false
-            } else {
-                true
-            }
+            !(*offset == text.len() && matches!(opportunity, BreakOpportunity::Mandatory))
```

Comment:
Is it just me or is the new condition harder to read?

I'd go at least with

`*offset != text.len() || !matches!(...)`